pymc3/backends/report.py

```diff
@@ -107,7 +107,7 @@ def _run_convergence_checks(self, trace, model):
             self._add_warnings([warn])
             return
 
-        from pymc3 import rhat, ess
+        from pymc3 import rhat, ess, _to_arviz
```

Comment:
I have used this `to_arviz` approach for readability, but I can change it to `from arviz import from_pymc3` if you prefer it

Reply:
Could `_to_arviz` be a place where to put storing of sampling metadata? (See https://github.com/arviz-devs/arviz/issues/1146)

Reply:
I definitely think it has to be added to `from_pymc3` so also to `_to_arviz`, is that the question?

Reply:
I didn't know `_to_arviz` existed, so to me it sounds like code duplication?

Reply:
I aliased `from_pymc3` to `_to_arviz`, I was worried calling a function called `from_pymc3` inside pymc code would be confusing. I guess it depends on the general degree of familiarity with ArviZ